csharp/ql/lib/semmle/code/csharp/dataflow/internal/DataFlowPrivate.qll

```diff
@@ -1131,7 +1143,10 @@ private module ArgumentNodes {
     SummaryArgumentNode() { FlowSummaryImpl::Private::summaryArgumentNode(_, this, _) }
 
     override predicate argumentOf(DataFlowCall call, int pos) {
-      FlowSummaryImpl::Private::summaryArgumentNode(call, this, pos)
+      exists(ArgumentPosition apos |
+        FlowSummaryImpl::Private::summaryArgumentNode(call, this, apos) and
+        apos.getPosition() = pos
+      )
     }
```

Comment:
Should this still be using an `int` for `pos`?

Reply:
No, this is purely an internal C# implementation predicate.